cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -122,12 +122,43 @@
     If you come across a script that makes a convincing case to be included in
     the head, then file an issue or PR to discuss including it.
 -->
-    {# Google Tag Manager #}
+
+    <!-- Google Optimize page-hiding snippet -->
+    <style>
```

Comment:
Honestly, it wouldn't surprise me. It's pretty black box stuff, and Google seems to be changing stuff frequently, especially in regards to Optimize. I'm not sure what we can do better, but maybe @cfarm's research will help us figure it out.